### ISSUE TITLE
Cannot resolve '@automattic/calypso-ui' in card, ribbon & screen-reader-text components

### ISSUE THREAD

**Issue by qafui** [Automattic/wp-calypso#36436]
#### Steps to reproduce
1. Clone repository
2. Using git bash as admin, run `npm install`
3. Run `npm start`

#### What I expected
Successful build

#### What happened instead
Build terminates after css build step. Might be related to commit #35811 

#### Browser / OS version
Windows 10 Enterprise

#### Screenshot / Video
```
ERROR in ./client/components/card/index.js
Module not found: Error: Can't resolve '@automattic/calypso-ui' in 'C:\Users\user\_code\wp-calypso\client\components\card'
 @ ./client/components/card/index.js 13:17-50
 @ ./client/blocks/gdpr-banner/index.jsx
 @ ./client/layout/logged-out.jsx
 @ ./client/controller/index.node.js
 @ ./client/my-sites/themes/index.node.js
 @ ./client/sections.js
 @ ./server/pages/index.js
 @ ./server/boot/index.js
 @ ./index.js

ERROR in ./client/components/card/compact.js
Module not found: Error: Can't resolve '@automattic/calypso-ui' in 'C:\Users\user\_code\wp-calypso\client\components\card'
 @ ./client/components/card/compact.js 13:17-50
 @ ./client/components/section-header/index.jsx
 @ ./client/my-sites/theme/theme-features-card.js
 @ ./client/my-sites/theme/main.jsx
 @ ./client/my-sites/theme/controller.jsx
 @ ./client/my-sites/theme/index.node.js
 @ ./client/sections.js
 @ ./server/pages/index.js
 @ ./server/boot/index.js
 @ ./index.js

ERROR in ./client/components/ribbon/index.js
Module not found: Error: Can't resolve '@automattic/calypso-ui' in 'C:\Users\user\_code\wp-calypso\client\components\ribbon'
 @ ./client/components/ribbon/index.js 13:17-50
 @ ./client/components/theme/index.jsx
 @ ./client/my-sites/themes/themes-site-selector-modal.jsx
 @ ./client/my-sites/theme/main.jsx
 @ ./client/my-sites/theme/controller.jsx
 @ ./client/my-sites/theme/index.node.js
 @ ./client/sections.js
 @ ./server/pages/index.js
 @ ./server/boot/index.js
 @ ./index.js

ERROR in ./client/components/screen-reader-text/index.js
Module not found: Error: Can't resolve '@automattic/calypso-ui' in 'C:\Users\user\_code\wp-calypso\client\components\screen-reader-text'
 @ ./client/components/screen-reader-text/index.js 13:17-50
 @ ./client/components/external-link/index.jsx
 @ ./client/components/environment-badge/index.jsx
 @ ./client/document/index.jsx
 @ ./client/document sync \.jsx$
 @ ./server/render/index.js
 @ ./server/pages/index.js
 @ ./server/boot/index.js
 @ ./index.js
```

**Comment by simison** [comment]
Does it work if you run `npm run build-packages` before `npm start`?

**Comment by qafui** [comment]
Thanks @simison but it doesn't work when I run `build-packages` before `npm start`. Tried it on a new clone of the repo as well. It works perfectly on my mac though.

**Comment by simison** [comment]
I'm not entirely sure what's the current state of Windows support in Calypso in general. @Automattic/team-calypso ?

**Comment by blowery** [comment]
Hmmmm... We're using an `npm` feature, `link:` refs, which boil down to symlinks. That _may_ not be supported on Windows...

Windows support of the Calypso server and build has always been pretty ad hoc. We'll happily accept patches that improve compatibility, but we don't really have anyone using it actively there. Docker, a VM running Linux, or WSL may be an answer though.

**Comment by qafui** [comment]
Thanks @blowery. Unfortunately I don't have the option of using Docker or a VM on my Windows laptop (work laptop). 

I did notice however, that the entry point and module in the `packages/components` folder, listed as `dist/cjs/index.js` and `dist/esm/index.js`, in the local `package.json` are missing. 

Excuse my ignorance of the build process but how exactly are the library components in this package to be found by the client components that reference them @Automattic/team-calypso @blowery ?

I'd like to contribute to fixing this if support for Windows is part of the project's priorities :)

**Comment by blowery** [comment]
> I did notice however, that the entry point and module in the packages/components folder, listed as dist/cjs/index.js and dist/esm/index.js, in the local package.json are missing.

does `npm run build-packages` make them appear?

**Comment by qafui** [comment]
> does `npm run build-packages` make them appear? Maybe that's the script that's not working on Windows...

I deleted the `dist` folder and run `npm run build-packages`. The dist folder was successfully built but the entry files didn't appear. I'll look into possible reasons why. 

Thanks
